plugins/inputs/internet_speed/internet_speed.go

```diff
@@ -12,13 +12,23 @@ import (
 // InternetSpeed is used to store configuration values.
 type InternetSpeed struct {
 	EnableFileDownload bool            `toml:"enable_file_download"`
+	Offset             string          `toml:"offset"`
```

Comment:
Use the power of Telegraf
```suggestion
	Offset             config.Duration          `toml:"offset"`
```